boards/arm/imxrt/imxrt1064-evk/src/imxrt_userleds.c

```diff
@@ -0,0 +1,100 @@
+/****************************************************************************
+ * boards/arm/imxrt/imxrt1064-evk/src/imxrt_userleds.c
+ *
+ *   Copyright (C) 2018 Gregory Nutt. All rights reserved.
+ *   Authors: Gregory Nutt <gnutt@nuttx.org>
+ *            David Sidrane <david_s5@nscdg.com>
```

Comment:
@jturnsek change ALL copyright from Greg and David to apache since they already sign SGA or ICLA. You can reference the original code to know which files can convert.